yml/apdb.yaml

```diff
@@ -2075,6 +2124,96 @@ tables:
   - name: bboxSize
     "@id": "#DiaSource.bboxSize"
     datatype: long
+  - name: pixelFlags
+    "@id": "#DiaSource.pixelFlags"
+    datatype: boolean
+    description: General failure flag, set if anything went wrong
+    fits:tunit:
+  - name: pixelFlags_bad
+    "@id": "#DiaSource.pixelFlags_bad"
+    datatype: boolean
+    description: Bad pixel in the Source footprint
+    fits:tunit:
+  - name: pixelFlags_cr
+    "@id": "#DiaSource.pixelFlags_cr"
+    datatype: boolean
+    description: Cosmic ray in the Source footprint
+    fits:tunit:
+  - name: pixelFlags_crCenter
+    "@id": "#DiaSource.pixelFlags_crCenter"
+    datatype: boolean
+    description: Cosmic ray in the Source center
```

Comment:
We should probably change all of the "in the source center" to "in the 3x3 region around the centroid.", since that matches the current PixelFlags docstrings that I rewrote a while back, and is much more helpful to a user.

That's what `meas_base/PixelFlags.cc` has had for a while now, so wherever you got these from, please check the descriptions there which are clearer in some cases.